examples/Dockerfile-poppler.demo

```diff
@@ -1,117 +1,50 @@
-
-RUN make -j5 install
-
-#=================================
-WORKDIR /polytracker/the_klondike/poppler-0.84.0/build/utils
-
-#Extract and instrument pdftotext, other poppler tools should work the same 
+COPY --from=poppler-sources /polytracker/the_klondike/poppler /polytracker/the_klondike/poppler
+
+# Install poppler dependencies
+RUN apt-get update && apt-get install -y libfreetype6-dev libfontconfig1-dev
+
+# Configure
+WORKDIR /polytracker/the_klondike/poppler
+RUN cmake -S . -B build \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DBUILD_SHARED_LIBS=OFF \
+	-DBUILD_GTK_TESTS=OFF \
+	-DBUILD_QT5_TESTS=OFF \
+	-DBUILD_CPP_TESTS=OFF \
+	-DENABLE_BOOST=OFF \
+	-DENABLE_CPP=OFF \
+	-DENABLE_GLIB=OFF \
+	-DENABLE_GTK_DOC=OFF \
+	-DENABLE_QT5=OFF \
+	-DENABLE_LIBOPENJPEG=unmaintained \
+	-DENABLE_CMS=none \
+	-DENABLE_LIBCURL=OFF \
+	-DENABLE_ZLIB=OFF \
+	-DENABLE_DCTDECODER=unmaintained \
+	-DENABLE_ZLIB_UNCOMPRESS=OFF \
+	-DWITH_JPEG=OFF \
+	-DWITH_PNG=OFF \
+	-DWITH_TIFF=OFF \
+	-DWITH_NSS3=OFF \
+	-DWITH_Cairo=OFF \
+	-DWITH_FONTCONFIGURATION_FONTCONFIG=OFF \
+	-DCMAKE_EXE_LINKER_FLAGS="-pthread"
+
+# Build
+RUN cmake --build build -j$(nproc)
+
+# Extract and instrument pdftotext, other poppler tools should work the same
+WORKDIR /polytracker/the_klondike/poppler/build/utils
 RUN get-bc -b pdftotext
 RUN ${CXX} --lower-bitcode -i pdftotext.bc -o pdftotext_track --libs /build_artifacts/libpoppler.a freetype fontconfig pthread --lists freetype fontconfig xml2 libz
-
-#RUN get-bc -b pdfinfo
-#RUN ${CXX} --instrument-bitcode -f pdfinfo.bc -o pdfinfo_track --output-bitcode pdfinfo_track.bc --libs /build_artifacts/libpoppler.a freetype fontconfig pthread
-
-#RUN get-bc -b pdftops 
-#RUN ${CXX} --instrument-bitcode -f pdftops.bc -o pdftops_track --output-bitcode pdftops_track.bc --libs /build_artifacts/libpoppler.a freetype fontconfig pthread
-
-#RUN get-bc -b pdfunite 
-#RUN ${CXX} --instrument-bitcode -f pdfunite.bc -o pdfunite_track --output-bitcode pdfunite_track.bc --libs /build_artifacts/libpoppler.a freetype fontconfig pthread
-
-#RUN get-bc -b pdfattach 
-#RUN ${CXX} --instrument-bitcode -f pdfattach.bc -o pdfattach_track --output-bitcode pdfattach_track.bc --libs /build_artifacts/libpoppler.a freetype fontconfig pthread
-
-#RUN get-bc -b pdfimages 
-#RUN ${CXX} --instrument-bitcode -f pdfimages.bc -o pdfimages_track --output-bitcode pdfimages_track.bc --libs /build_artifacts/libpoppler.a freetype fontconfig pthread
```

Comment:
Any particular reason we don't instrument these? I suggest that we at least include pdftops, that would allow for a comparison with mutool .ps export.

Reply:
> Any particular reason we don't instrument these?

I figured we weren't using them and the instrumentation command was the same so I got rid of them.

> I suggest that we at least include pdftops, that would allow for a comparison with mutool .ps export.

Agreed!